src/main/settings.py

```diff
@@ -58,8 +59,49 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "author.middlewares.AuthorDefaultBackendMiddleware",
 ]
 
+
+CELERY_BROKER_URL = "amqp://guest:guest@rabbitmq"
+IMPORT_EXPORT_CELERY_INIT_MODULE = "main.celery"
+
+# For files
+FILE_UPLOAD_PERMISSIONS = 0o777
```

Comment:
Over het algemeen wordt 755 of misschien 775 aangehouden. 777 is vrijwel altijd een noodgreep en wordt niet aangeraden.